app/components/tf-fields/label.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-ticketfly-fields/components/tf-fields/label';
```

Comment:
We might not, but there's no way to have private contextual components in Ember addons. Right now, we have to import them as well.